src/Makefile.defs

```diff
@@ -1017,7 +1017,7 @@ ifeq		($(CC_NAME), gcc)
 				#common stuff
 				CFLAGS+=$(CC_OPT) -funroll-loops  -Wcast-align $(PROFILE)
 			#if gcc 4.5+
-ifeq			($(CC_SHORTVER),$(filter $(CC_SHORTVER),4.2+ 4.5+ 5.0+ 6.0+ 7.0+ 8.0+ 9.0+))
+ifeq			($(CC_SHORTVER),$(filter $(CC_SHORTVER),4.2+ 4.5+ 5.0+ 6.0+ 7.0+ 8.0+ 9.0+ 15.0+))
```

Comment:
This should not be needed, since the replacement above turns `1[0-5]....` to `9.0+`.

Reply:
Without it I get:
```
$ rm src/config.mak  ; make config
make -C src/ config 
make[1]: Entering directory '/git/voip/kamailio/src'
target architecture <x86_64>, host architecture <x86_64>
Makefile.defs:1062: You are using an old and unsupported gcc version  (15.0+), compile at your own risk!
making config...
rm -f modules.lst
make --no-print-directory modules.lst
saving modules list...
make[1]: Leaving directory '/git/voip/kamailio/src'
```
If you cannot reproduce this, where should I look at?

Reply:
```
sed -e 's/5\.[0-9]\..*/5.0+/' -e 's/5\.[0-9]$$/5.0+/' 
```
converts 15.1.1 to 15.0+, and `-e 's/1[0-5]\.[0-9]\..*/9.0+/' -e 's/1[0-5]\.[0-9]$$/9.0+/` does not apply.  Adding ^ in `/^5` does result `9.0+`.

Reply:
I added `^` to all the sed expressions.